README.md

```diff
@@ -1,3 +1,75 @@
-# pos
+# POS - Point Of Sales
 
-96Boards Point-of-Sale System
+# Table of Contents
+
+- [1) Hardware](#1-hardware)
+   - [1.1) Hardware Requirements](#11-hardware-requirements)
+   - [1.2) Hardware Setup](#12-hardware-setup)
+- [2) Software](#2-software)
+   - [2.1) Client Build Environment Setup](#21-client-build-environment-setup)
+   - [2.2) Database Server Setup](#22-api-keys)
+   - [2.3) Running the Application](#23-running-the-application)
+
+# 1) Hardware
+
+## 1.1) Hardware Requirements
+
+- [Any 96Boards CE](https://www.96boards.org/products/ce/) x3
```

Comment:
What does x3 mean? 3 boards? In that case use below:

[Any 96Boards CE](https://www.96boards.org/products/ce/) - 3 Nos